static/src/stylesheets/head.hosted-amp.scss

```diff
@@ -12,6 +12,8 @@
 @import 'amp/_type';
 @import 'amp/_fonts';
 @import 'amp/_helpers';
+@import 'amp/_content';
```

Comment:
Hmm. I was being safe in case there were more styles that were missing, I think there's only one or two we need though.